proto/compliance/tx.proto

```diff
@@ -85,4 +86,33 @@ message MsgProvisionModel {
 message MsgProvisionModelResponse {
 }
 
+message MsgUpdateComplianceInfo {
+  string creator = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.moretags) = "validate:\"required\""];
```

Comment:
The following fields should be non-editabe (and excluded from this message):
- softwareVersionCertificationStatus. It must be changed via Certify/Revoke/Provision calls.
- softwareVersionString (it must be the same as the one from the corresponding ModelVersion, it's checked during creation. As softwareVersionString is immutable in ModelVersion, it must be immutable here as well).
- CertificationType (as it's one of key values). If it needs to be changed, Certify/Revoke/Provision calls should be used.